azure-core.gemspec

```diff
@@ -40,11 +40,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency('rake',                '~> 10.0')
   s.add_development_dependency('timecop',             '~> 0.7')
   s.add_development_dependency('bundler',             '~> 1.11')
-  if RUBY_VERSION < "2.1.0"
```

Comment:
No it is no longer picked up from the file. We now updated the `Readme.md` so that user can resolve the issue manually. This is a limitation caused by Nokogiri and we are not able to provide any solution or workaround for the issue as long as Nokogiri is one of our dependency. It is a common case for projects that depends on it.